ios/Components/InstantComponent.swift

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+
+import Adyen
+import Foundation
+import PassKit
+import React
+
+@objc(AdyenInstant)
+internal class InstantComponent: BaseModule {
+    
+    private var currentComponent: PresentableComponent?
+    private var currentPaymentComponent: PaymentComponent?
+    private var actionHandler: AdyenActionComponent?
+    
+    @objc
+    override static func requiresMainQueueSetup() -> Bool { true }
+    override func stopObserving() {}
+    override func startObserving() {}
+    override func supportedEvents() -> [String]! { super.supportedEvents() }
+
+    @objc
+    func hide(_ success: NSNumber, event: NSDictionary) {
+        DispatchQueue.main.async {
+            self.currentComponent?.finalizeIfNeeded(with: success.boolValue)
+            UIApplication.shared.keyWindow?.rootViewController?.dismiss(animated: true)
```

Comment:
just grabbing the `rootViewController`? :-D , that's a bit dangerous, because it will dismiss everything even if they don't belong to the SDK

Reply:
good point.. 😅